sass/blocks/_story-hero.scss

```diff
@@ -0,0 +1,83 @@
+/*--------------------------------------------------------------
```

Comment:
Why do we have custom CSS here for our Story Hero? Is this because we haven't updated the UDS Bootstrap css yet?

Reply:
Yes, that was the case when we started this pull request. It's been integrated into UDS-Bootstrap as of about a week ago, but I'm not 100% sure if we were able to update the NPM package in our theme to integrate it from the source.

If that happens, this SASS partial can be outright deleted. It's an exact duplicate of what was submitted to Bootstrap.

Reply:
Double checked. The story hero SASS has not been included in the Bootstrap package that's a part of this PR. 

I'm fairly confident that when we merge #192 that it will be there. When that merge happens the correct thing to do would be to remove the `blocks/story-hero` partial completely.